ffi/derive/src/lib.rs

```diff
@@ -299,6 +307,25 @@ fn derive_try_from_ffi_for_opaque_item(name: &Ident) -> TokenStream2 {
                 Ok(store)
             }
         }
+
+        impl<'itm> iroha_ffi::owned::TryFromReprCVec<'itm> for #name {
+            type Source = iroha_ffi::slice::SliceRef<'itm, <Self as iroha_ffi::TryFromReprC<'itm>>::Source>;
+            type Store = ();
+
+            unsafe fn try_from_repr_c(
+                source: Self::Source,
+                _: &'itm mut <Self as iroha_ffi::owned::TryFromReprCVec<'itm>>::Store,
+            ) -> Result<Vec<Self>, iroha_ffi::FfiResult> {
+                let slice = source.into_rust().ok_or(iroha_ffi::FfiResult::ArgIsNull)?;
```

Comment:
I have added `FfiResult::ConversionFailed`, however for this particular case  `FfiResult::ArgIsNull` is fine, because only case in which `slice.into_rust()` return `None` is when `slice` is `null`.